allennlp/semparse/contexts/table_question_knowledge_graph.py

```diff
@@ -61,6 +61,82 @@
         **MONTH_NUMBERS,
         }
 
+
+class ColumnTyper():
```

Comment:
You don't need parens at the end of the class name.

Reply:
I'm thinking of renaming QuestionEntityExtractor to "TableQuestionsContext" and add column typing as a public method inside that. What do you think?

Reply:
Yeah, `TableQuestionContext` seems fine.  Make sure the docstring says what's different about it from `TableQuestionKnowledgeGraph`.